test/PublishCommand.js

```diff
@@ -460,6 +486,52 @@ describe("PublishCommand", () => {
   });
 
   /** =========================================================================
+   * NORMAL - SKIP GIT PUSH AND SKIP NPM
+   * ======================================================================= */
+
+  describe("normal mode with --skip-git-push and --skip-npm", () => {
```

Comment:
Unless there is some sort of interaction between `--skip-git-push` and `--skip-npm`, I'm not sure how important this test case is. We've already established what `--skip-npm` does, after all, and `--skip-git-push` doesn't change anything about that?